content/support.md

```diff
@@ -14,12 +14,3 @@ Learn more in our [support policy](https://github.com/kedacore/governance/blob/m
 Here's an overview of all vendors that provide KEDA as part of their offering/product and provide support for it: 
 
 {{< support >}}
-
-## Kubernetes support
```

Comment:
That's fine for me, but we might want to add a link in the OSS support section to /latest/operate/cluster

Reply:
Fixed